docs/development/getting-started.rst

```diff
@@ -58,6 +58,10 @@ you stay up-to-date with our repository:
 Configure the development environment
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+.. note::
+   Do not use venv/virtualenv for configuring the development environment.
+   Docker creates and remove venv/virtualenv as needed.
```

Comment:
hello @raghav-ys, thanks for taking the time to improve the docs!

Personally, I would actually prefer the wording of @brainwane mentioned in the original issue #6783, for I find it more developer friendly. Reading a direct "Do not" made me stall a bit 😆
More importantly, the suggested text in the original issue is more accurate, for Docker does not create/remove stuff on its own.

Keep up contributing!